storage/watchable_store_test.go

```diff
@@ -222,3 +239,118 @@ func TestUnsafeAddWatching(t *testing.T) {
 		}
 	}
 }
+
+func TestNewMapWatchingToEventMap(t *testing.T) {
+	k0, k1, k2 := []byte("foo0"), []byte("foo1"), []byte("foo2")
+	v0, v1, v2 := []byte("bar0"), []byte("bar1"), []byte("bar2")
+
+	testWatchings := []*watching{
```

Comment:
just call this watchings instead of testwatchings. When the context is already test, we should avoid repeat it again in variable.